lib/openstudio-standards/standards/Standards.FluidCooler.rb

```diff
@@ -36,7 +36,7 @@ def fluid_cooler_apply_minimum_power_per_flow(fluid_cooler)
     search_criteria['template'] = template
 
     # Closed cooling towers are fluidcooler objects.
-    search_criteria['equipment_type'] = 'Closed Cooling Tower'
+    search_criteria['equipment_type'] = 'Dry Cooler'
```

Comment:
Perhaps this needs to be an argument?  Per 90.1-2013 Table 6.8.1-7, there are values for '...closed-circuit cooling towers' and 'air cooled condensers'.  Maybe this is an additional property for the fluid cooler object?